src/components/VerticalListMenu/examples/1.basic.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import VerticalListMenuStateless from '../VerticalListMenu';
+import { buildStatefulComponent } from '../../../util/state-management';
+
+const VerticalListMenu = buildStatefulComponent(VerticalListMenuStateless);
+
+export default React.createClass({
+	render() {
+		return (
+			<VerticalListMenu style={{ width: '250px' }}>
```

Comment:
The React docs themselves highlight the built-in feature of automatically converting a number value to a string value ending in `px`: https://facebook.github.io/react/tips/style-props-value-px.html